assets/src/js/admin/reports/components/chart/utils/index.js

```diff
@@ -81,14 +82,20 @@ export function createConfig (type, data) {
 
     // Setup yAxes to begin at zero if chart is 'line' or 'bar'
     if (type === 'line' || type === 'bar') {
+
+		const yAxes = data.datasets.map((dataset, index) => {
+			return {
+				id: 'y-axis-' + index,
```

Comment:
@henryholtgeerts you[ javascript template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). I will create an issue to auto-fix this: https://eslint.org/docs/rules/prefer-template

cc: @JasonTheAdams @DevinWalker

Reply:
Good catch! Altered this line to use template literals, ref: https://github.com/impress-org/givewp/pull/4412/commits/b1f7c802752ec959fddb2cab6253ef517f224f4f

Reply:
Also: https://github.com/impress-org/givewp/pull/4412/commits/428b70c03489f2b8f9c67de82304874d61994bca